File: golang/99.go
package main

import "fmt"

func main() {
	for i := 99; i > 0; i-- {
		fmt.Println(i, "bottles of beer on the wall,", i, "bottles of beer")
		fmt.Println("Take one down and pass it around,", i-1, "bottles of beer on the wall")
	}
}

File: perl/99.pl
#!/usr/bin/env perl

use strict;
use warnings;

for (my $i = 99; $i > 0; $i--) {
    print qq($i bottles of beer on the wall, $i bottles of beer\n);
    print qq(Take one down and pass it around, @{[$i - 1]} bottles of beer on the wall\n);
}